applications/Awesome-Polka.md

```diff
@@ -223,7 +223,7 @@ This project is planned as 1 milestone, it will be completed in one months.
 | **0c.** | Testing and Testing Guide | Core functions will be fully covered by comprehensive unit tests to ensure functionality and robustness. In the guide, we will describe how to run these tests. |
 | **0d.** | Docker | I will provide a Dockerfile(s) that can be used to test all the functionality delivered with this milestone. |
 | **0e.** | Article | I will publish an **article**/workshop that explains what was done/achieved as part of the grant. |
-| **1.** | UI & UX Development | **Home Page:** Visually appealing and user-friendly homepage which includes latest articles, popular categories and many more <br /> **Projects Page:** Showcase page with detailed filtering and search infrastructure <br/>**Project Detail Page:** Project details page to display detailed information about project with several modules like explained in the solution <br />**Articles Page:** Articles page to showcase informative articles related to the subject <br />**Article Detail Page:** Detail page for individual articles to display their full content <br />**Ranking Page:** Ranking page based on votes and token stats which is supported by CoinGecko api to showcase popular projects and articles <br/> **Learn Page:** Useful resources to learn about polkadot ecosystem </br/> **UX Improvements & Testing:** I will work to improve the user experience by ensuring that the user interface is fully compatible with mobile and tablet devices. |
```

Comment:
Did the rules about `br` change? Last time anything but `br /` didn't compile.

Reply:
No still true. Fixed now.